systems/summit_user_guide.rst

```diff
@@ -408,6 +408,31 @@ using the modules system:
     pgi/19.5
     pgi/19.7
 
+Compiling for Projects in the Moderate Enhanced Security Enclave 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Moderate Enhanced projects need to compile code on a batch node rather than the moderate-enhanced login node because the
```

Comment:
Maybe word this so it notes the temporary nature.

Reply:
The last line does say it is temporary. I'll pull it out as a .. note::